src/supermarket/Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 # Ruby version isn't specified here anymore, as it's pinned in Omnibus
 # https://github.com/chef/omnibus-supermarket/blob/master/config/projects/supermarket.rb
 
-gem 'fieri', path: '../fieri'
+gem 'fieri', path: 'engines/fieri'
```

Comment:
The tip of the ice berg. With fieri's files now located under the supermarket application root, the app root can now be the top-most directory included in a habitat package or container. Build and runtime no longer have to reach outside the app root tree.